package.json

```diff
@@ -9,13 +10,13 @@
     ">10%"
   ],
   "dependencies": {
-    "@activeadmin/activeadmin": "^3.3.0",
+    "@activeadmin/activeadmin": "4.0.0-beta15",
```

Comment:
Note: we seemingly cannot do `^4.0.0-beta15`, because I think that `pnpm` (or NPM, or someone) will consider `4.0.0-beta9` to be the highest available version and will install that, instead.

(It seems that bundler is smarter, and realizes that 4.0.0-beta15 is a higher version than 4.0.0-beta9, so we are specifying the gem with `>= 4.0.0.beta15`.)